oss-internship-2020/jsonnet/examples/jsonnet_base_example.cc

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdlib>
+#include <iostream>
+
+#include "jsonnet_base_sandbox.h"
+
+int main(int argc, char* argv[]) {
+  google::InitGoogleLogging(argv[0]);
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+
+  if (!(argc == 3)) {
+    std::cerr << "Usage:\n"
+              << basename(argv[0]) << " absolute/path/to/INPUT.jsonnet"
+              << " absolute/path/to/OUTPUT\n";
+    return EXIT_FAILURE;
+  }
+
+  std::string in_file(argv[1]);
+  std::string out_file(argv[2]);
+
+  // Initialize sandbox.
+  JsonnetBaseSandbox sandbox(in_file, out_file);
+  absl::Status status = sandbox.Init();
+  CHECK(status.ok()) << "Sandbox initialization failed: " << status;
```

Comment:
Error handling: Like in https://github.com/google/sandboxed-api/blob/master/oss-internship-2020/pffft/main_pffft_sandboxed.cc#L90, I suggest to refactor this to have a `JsonnetMain()` function that returns an `absl::Status` and to use the `SAPI_RETURN_IF_ERROR` and `SAPI_ASSIGN_OR_RETURN` macros for error handling.
That way, you don't have to have `CHECK()`s everwhere can can handle errors in a single location.